File: Frontend/trackr-paciente/src/app/shared/dtos/perfil/informacion-general-dto.ts
import {ExpedientePadecimientoDto} from '../seguridad/expediente-padecimiento-dto'

export interface InformacionGeneralDto {

    nombre: string;
    apellidoPaterno: string;
    apellidoMaterno: string;
    fechaNacimiento: Date;
    idGenero: number;
    peso: number;
    cintura: number;
    estatura: number;
    correo: string;
    telefonoMovil: string;
    idPais: number;
    idEstado: number;
    idMunicipio: number;
    idLocalidad: number;
    idColonia: number;
    codigoPostal: string;
    calle: string;
    entreCalles: string;
    numeroInterior: string;
    numeroExterior: string;

    padecimientos: ExpedientePadecimientoDto[];
}

File: Backend/TrackrAPI/TrackrAPI/Dtos/Perfil/InformacionGeneralDTO.cs
using TrackrAPI.Dtos;
using TrackrAPI.Dtos.GestionExpediente;
using TrackrAPI.Models;

namespace TrackrAPI.Dtos.Perfil
{
    public class InformacionGeneralDTO
    {
        public string Nombre { get; set; }
        public string ApellidoPaterno { get; set; }
        public string ApellidoMaterno { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public int IdGenero { get; set; }
        public decimal Peso { get; set; }
        public decimal Cintura { get; set; }
        public decimal Estatura { get; set; }
        public string Correo { get; set; }
        public string CorreoPersonal { get; set; }
        public string TelefonoMovil { get; set; }
        public bool? CorreoConfirmado { get; set; }
    }
}